smoke-test/tests/cli/user_groups_cmd/test_user_cmd.py

```diff
@@ -11,7 +11,7 @@
 from datahub.entrypoints import datahub
 from tests.consistency_utils import wait_for_writes_to_sync
 
-runner = CliRunner(mix_stderr=False)
+runner = CliRunner()
```

Comment:
We should probably have a DataHubCliRunner wrapper

tbh I’m surprised we don’t already have one